official-templates/fast-stable-diffusion/Dockerfile

```diff
@@ -8,6 +8,7 @@ WORKDIR /
 ENV SHELL=/bin/bash \
     PYTHONUNBUFFERED=1 \
     DEBIAN_FRONTEND=noninteractive
```

Comment:
You are missing a slash here.

Reply:
Thank you for the suggestion, i didn't see that